app/javascript/Components/table/table.jsx

```diff
@@ -119,6 +143,11 @@ export default function Table({columns, data, noDataText, initialState}) {
                     );
                   })}
                 </div>
+                {row.getIsExpanded() && (
+                  <tr>
```

Comment:
I don't really think it makes sense to use `<tr>`/`<td>` elements here, since this isn't a "real" table. Instead, please just use a `<div>` that wraps the subcomponent, e.g.

```jsx
<div>{renderSubComponent({row})</div>
```

Unless that looks bad for some reason (provide screenshot).